app/moved_urls.yml

```diff
@@ -14,33 +15,35 @@
 /gateway/VERSION/install-and-run/migrate-ce-to-ke/: "/gateway/VERSION/migrate-ce-to-ke/"
 /gateway/VERSION/install-and-run/upgrade-enterprise/: "/gateway/VERSION/upgrade/"
 /gateway/VERSION/install-and-run/upgrade-oss/: "/gateway/VERSION/upgrade/"
-/gateway/2.8.x/get-started/quickstart/: "/gateway/VERSION/get-started/"
-/gateway/2.8.x/get-started/quickstart/configuring-a-service/: "/gateway/VERSION/get-started/services-and-routes/"
-/gateway/2.8.x/get-started/quickstart/configuring-a-grpc-service/: "/gateway/VERSION/kong-plugins/configuring-a-grpc-service"
+/gateway/VERSION/get-started/quickstart/: "/gateway/VERSION/get-started/"
+/gateway/VERSION/get-started/quickstart/configuring-a-service/: "/gateway/VERSION/get-started/services-and-routes/"
+/gateway/VERSION/get-started/quickstart/configuring-a-grpc-service/: "/gateway/VERSION/kong-plugins/configuring-a-grpc-service"
```

Comment:
I think this one isn't working because when I click "latest documentation", it's still taking me to the 2.8.x page. 
 This is the page I was looking at: https://deploy-preview-4373--kongdocs.netlify.app/gateway/2.8.x/get-started/quickstart/configuring-a-grpc-service/